low-level/framework/base/sspl_constants.py

```diff
@@ -212,8 +212,18 @@
 except Exception as err:
     logger.debug(f"Error in getting setup information of server and storage type : {err}")
     print(f"Error in getting setup information of server and storage type : {err}")
-    storage_type = 'virtual'
-    server_type = 'virtual'
+    result = subprocess.Popen(['consul',
```

Comment:
code in ssppl_constants.py file needs to be avoided, atleast lets not add new one. Lets have default values hardcoded here. Configurable values would come from conf file

Reply:
Now I separated the coding work from sspl_constants.py and kept the logic in new file data_provider.py(fetch values form consul/file).
sspl_constants.py gets storage_type and server_type from it.